### DISCUSSION TITLE
Iteration @current/@next/@previous problems near iteration boundaries

### DISCUSSION THREAD

**Discussion by mseeley** [community/community#59926] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

## Context

It's approximately 4:30PM HST. In the last 30 minutes or so the tabs filtering by iteration `@current`, `@previous`, and `@next` have started to fail. We've run into this several times over our previous iterations.

## Screenshots

For context, today is the end of Iteration 11.

`@current` iteration is empty.

<img width="1299" alt="image" src="https://github.com/community/community/assets/30019/4d59d4e9-fdae-4473-815b-e726db68f310">

`@next` iteration is empty.

<img width="1445" alt="image" src="https://github.com/community/community/assets/30019/84d7538e-b148-406e-88fa-bea5bc6b6444">

`<@current` shows as invalid. Notice that this tab understands `@current`:

<img width="1449" alt="image" src="https://github.com/community/community/assets/30019/7667e970-885e-4ffb-9f5b-6c0a56337533">

<img width="634" alt="image" src="https://github.com/community/community/assets/30019/5f4b0578-11f8-4aff-ac9f-e2be1ec05128">

## Observations

- There are hints that this is timezone/time based.
- My colleagues in Japan do not see the same behavior I see here in Hawaii. 
- The project tabs displayed correctly earlier today and will again tomorrow.

Please help. 🙇 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by svix-onelson** [comment]
I'm assuming this is the same issue I came here to report 😅.

I've seen that project boards filtered by `@current` are empty on Sunday night (the last day of our iteration), whereas `@previous` and `@next` show the expected issues.

**Comment by mseeley** [comment]
`@current` is working as expected today across the screenshots shared yesterday.

**Comment by mseeley** [comment]
Our current iteration is 15. The project page was working fine earlier today. Now it's 6PM HST and it's broken again.

`@current` is broken again; notice the squiggly

![SCR-20230813-pssw](https://github.com/community/community/assets/30019/ef278930-eae2-45d0-a473-1155f184df3b)

Iteration 15 is empty. Please trust me, it's not.

![SCR-20230813-psyi](https://github.com/community/community/assets/30019/b0df0e79-544f-4b11-898e-33c82472d5fe)

**Comment by mseeley** [comment]
Hi @j0siepy, I found you over in https://github.com/orgs/community/discussions/63402; I'm sorry for cold-calling. Is there a chance you can help me find a GitHub colleague to triage?

**Reply by j0siepy** [reply]
No worries! I'll ping the team working on iterations.

**Reply by mseeley** [reply]
Hi @azenMatt, @queenofcorgis, @j0siepy. I apologize for the noise. I found your user names as creators of several pinned [projects-and-issues](https://github.com/orgs/community/discussions/categories/projects-and-issues) threads.

Will any of you please help this issues reach the team working on iterations? We run into this when trying to plan the next iteration on the last day of the current iteration. It seems like there are troubles dealing with this, at least from the Hawaii timezone. Island time curse?

We are rescheduling our meetings to work around this behavior although, please, may I ask for help routing this issue to the appropriate triage crew? I'm available to provide any input practical.

**Reply by azenMatt** [reply]
Hi @mseeley - thanks for reporting this - it's now with our triage crew. I'l keep you posted

**Reply by azenMatt** [reply]
Hi @mseeley - thank you for your patience with this one - it has now been fixed!

**Comment by mseeley** [comment]
Time on my computer is Sun Aug 27 at 7:02pm HST. Iteration 15 has been archived:

<img width="1068" alt="image" src="https://github.com/community/community/assets/30019/b548a2bf-016b-47a1-9a41-1790d2ff2f65">

Iteration 16 is "current".

<img width="1059" alt="image" src="https://github.com/community/community/assets/30019/de88f242-23e2-4add-adfd-5a08561350b5">

But, `@current` is still broken; see red squiggly.

<img width="1493" alt="image" src="https://github.com/community/community/assets/30019/26ac15d4-2a73-493e-bc56-dab915132ec0">

The current iteration tab shows no work from Iteration 15.

<img width="1421" alt="image" src="https://github.com/community/community/assets/30019/0fa43345-8fcb-4a75-8227-da22acc2d1b6">
